templates/zerver/documentation_main.html

```diff
@@ -8,9 +8,9 @@
 
 {% block portico_content %}
 <div class="app help terms-page inline-block{% if page_is_help_center %} help-center{% endif %}{% if page_is_api_center %} api-center{% endif %}">
-    <div class="sidebar">
+    <div class="{{ sidebar_class }}">
         <div class="content">
-            <h1><a href="{{ doc_root }}" class="no-underline">Home</a></h1>
+            <h1><a href="{{ doc_root }}" class="no-underline">Contents</a></h1>
```

Comment:
Hmm, this name doesn't describe /api correctly; do we need add a sidebar include for the /api page as well?